### PR TITLE
[docs] Split charts overview and getting started in distinct pages

### PR DESCRIPTION
**Pull request by alexfauquette**
Built on top of #10710

In this PR:

- Split `overiew` in `overview`/`getting-started`
- Put subcategory to distinguish components and shared fetures
- Modify the title `Tooltip` to `Tooltip & Highlights`
- Add a short explanation about the docs structure in the overview
- Add a callout about the Next.js issue

## before/after

image

### CODE REVIEW

File: docs/data/charts/composition/composition.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+The first one requires to provide `width` and `height` props.
+Whereas the second one will compute the undefined dimension based on the size of its parent element.
+
+:::warning
+The parent element should have intrinsic dimensions.
+If the parent dimension relies on its content, the responsive charts will not render.
+:::
+
+The next demo allows switching between a chart using `<ChartContainer />` with `width` (resp. `height`) props set to 500 (resp. 300),
+and a chart using `<ResponsiveChartContainer />`.
```
Comment (michelengelen) [inline docs/data/charts/composition/composition.md:38]:
```suggestion
The next demo allows switching between a chart using `<ChartContainer />` with `width` set to `500` and `height` set to `300`,
and a chart using `<ResponsiveChartContainer />`.
```

**Comment by alexfauquette** [comment]
@michelengelen I created this draft PR to keep #10710 as simple as possible and have the follow-up (this PR) ready. 

The goal is to have one PR for content modification, and another one for file modification

I will do the rebase, and other stuff after the first one get merged

File: docs/data/charts/getting-started/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+With a single component or by composing sub-components.
+
+### Single charts
+
+For common use cases, the single component is the recommended way.
+
+Those components' name ends with "Chart" and only require the `series` prop, describing the data to render.
+
+They also have plenty of other props to customize the chart behavior.
+
```
Comment (alexfauquette) [inline docs/data/charts/getting-started/getting-started.md:52]:
```suggestion
```

File: docs/data/charts/getting-started/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+{{"demo": "SimpleCharts.js"}}
+
+### Multiple charts
```
[the remark this answers is not in the document]
Comment (alexfauquette) [reply docs/data/charts/getting-started/getting-started.md:55]: What about "Composed Charts" since the technic is called composition?

**Comment by alexfauquette** [comment]
I found why it was such a mess. I opened the PR was pointing to the next branch but was written on top of the master one.

Sorry @samuelsycamore and @michelengelen  for the bad experience. I included all your comments about composition page in #10710 and merged it

File: docs/data/charts/getting-started/getting-started.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (alexfauquette) [reply docs/data/charts/getting-started/getting-started.md:52]:
```suggestion
### Composed Charts
```

**Pull request merged**